core/src/main/scala/org/apache/spark/status/AppStatusListener.scala

```diff
@@ -280,6 +284,12 @@ private[spark] class AppStatusListener(
   private def updateBlackListStatus(execId: String, blacklisted: Boolean): Unit = {
     liveExecutors.get(execId).foreach { exec =>
       exec.isBlacklisted = blacklisted
+      if (blacklisted) {
+        appStatusSource.foreach{_.BLACKLISTED_EXECUTORS.inc(1)}
```

Comment:
Nit: you don't need a block here, just parens. Same for several calls below.

Reply:
yeah I thought intellij complained, will fix.